### PR TITLE
Update palette tokens for Scrim styling

### PR DESCRIPTION
**Pull request by libertymayc**

### CODE REVIEW

File: .changeset/tasty-maps-jump.md [hunk 1]
```diff
@@ -129,10 +129,8 @@ Status: replace emphasis tokens with 'emphasize' type
 Palette: token renaming emphasis to variants; color is the same where not stated
 
 ````diff
```
Comment (libertymayc) [inline .changeset/tasty-maps-jump.md:131]: even though this was done in that pr, should i just delete the notes about it here so its not confusing? @joshwooding

File: packages/theme/stories/characteristics/overlayable.stories.mdx [hunk 2]
```diff
@@ -80,7 +61,7 @@ Components that may form a hierarchy and stack upon or beneath other elements of
 - `Mega Menu`
 - `Overlay`
 - `Preferences Dialog` (inherited)
-- `Scrim`
+- `backdrop`
```
Comment (joshwooding) [inline packages/theme/stories/characteristics/overlayable.stories.mdx:64]:
```suggestion
- `Scrim`
```

File: packages/core/src/scrim/Scrim.css [hunk 3]
```diff
@@ -20,7 +11,7 @@
   position: fixed;
   align-items: center;
   justify-content: center;
-  background: var(--uitkScrim-background, var(--scrim-background));
+  background: var(--uitkScrim-background, var(--uitk-overlayable-primary-background));
```
Comment (joshwooding) [inline packages/core/src/scrim/Scrim.css:14]:
```suggestion
  background: var(--uitkScrim-background, var(--uitk-overlayable-background));
```

File: packages/core/src/scrim/Scrim.tsx [hunk 4]
```diff
@@ -225,8 +220,7 @@ export const Scrim = forwardRef<HTMLDivElement, ScrimProps>(function Scrim(
     <div
       aria-modal={!enableContainerMode}
       className={classnames(className, withBaseName(), {
-        [withBaseName(`containerFix`)]: enableContainerMode,
-        [withBaseName(variant)]: variant === "secondary",
+        [withBaseName(`containerFix`)]: enableContainerMode
```
Comment (joshwooding) [inline packages/core/src/scrim/Scrim.tsx:223]:
```suggestion
        [withBaseName("containerFix")]: enableContainerMode
```

**Review by joshwooding — CHANGES_REQUESTED** [review]
See comments

**Review by joshwooding — APPROVED** [review]

**Review by joshwooding — APPROVED** [review]

**Pull request merged**